source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -71,6 +71,9 @@ def _normalizeControlField(self, attrs):
 class ChromeVBuf(GeckoVBuf):
 	TextInfo = ChromeVBufTextInfo
 
+	# selecting with IAccessibleTextSelectionContainer is currently broken in Chromium.

```

Comment:
Has this been reported to Chromium? If it hasn't been are you planning to do so? If it has can this comment be updated, to reference a Chromium bug?